pkg/runtime/runtime.go

```diff
@@ -790,7 +803,8 @@ func (a *DaprRuntime) publishMessageGRPC(msg *pubsub.NewMessage) error {
 func (a *DaprRuntime) initActors() error {
 	actorConfig := actors.NewConfig(a.hostAddress, a.runtimeConfig.ID, a.runtimeConfig.PlacementServiceAddress, a.appConfig.Entities,
 		a.runtimeConfig.GRPCPort, a.appConfig.ActorScanInterval, a.appConfig.ActorIdleTimeout, a.appConfig.DrainOngoingCallTimeout, a.appConfig.DrainRebalancedActors)
-	act := actors.NewActors(a.stateStore, a.appChannel, a.grpc.GetGRPCConnection, actorConfig)
+	// TODO: provide a way to specify the actor state store, currently picking the first configured/default store
```

Comment:
This needs to be addressed.

We discussed reading a metadata property from the component spec which is `actorStore: true` to indicate this is the state store to be used for actors. If multiple stores exist, we pick the first one as default.

Reply:
thanks @yaron2  for reviewing the code. I have pushed the suggested changes.